### PR TITLE
docs: Remove coroutine/DPP_CORO experimental warnings

### PR DESCRIPTION
**Pull request by DJOetzi**
Removed notices of coroutines being an experimental feature.

## Documentation change checklist

- [x] My documentation changes follow the [docs style guide](https://dpp.dev/docs-standards.html) and any code examples follow the [coding style guide](https://dpp.dev/coding-standards.html).
- [x] I tested that my change works before raising the PR (via running `doxygen`, and testing examples).
- [x] I have not moved any existing pages or changed any existing URLs without strong justification as to why.
- [x] I have not generated content using AI or a desktop utility such as grammarly.

### CODE REVIEW

**Review by braindigitalis — APPROVED** [review]

File: docpages/example_programs/using_coroutines.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
 \page using-coroutines Using Coroutines
 
-\include{doc} coro_warn.dox
```
Comment (Jaskowicz1) [inline docpages/example_programs/using_coroutines.md:3]: The `coro_warn.dox` needs to stay. People needs to know that Coro is only supported on certain compiler versions. Just remove the last part of the `coro_warn.dox` :)

**Review by Jaskowicz1 — CHANGES_REQUESTED** [review]
You're missing https://github.com/brainboxdotcc/DPP/blob/ced36fd7652ed24ec3784cbc150722a1e5f6456a/include/dpp/coro/job.h#L48

https://github.com/brainboxdotcc/DPP/blob/ced36fd7652ed24ec3784cbc150722a1e5f6456a/include/dpp/coro/async.h#L103

https://github.com/brainboxdotcc/DPP/blob/ced36fd7652ed24ec3784cbc150722a1e5f6456a/library/CMakeLists.txt#L292 (just remove the word "experimental" here, keep the `message()`).

https://github.com/brainboxdotcc/DPP/blob/ced36fd7652ed24ec3784cbc150722a1e5f6456a/include/dpp/coro/task.h#L87

https://github.com/brainboxdotcc/DPP/blob/ced36fd7652ed24ec3784cbc150722a1e5f6456a/include/dpp/coro/coroutine.h#L67

**Comment by Jaskowicz1** [comment]
Any update on this?

**Comment by DJOetzi** [comment]
> You're missing
> 
> https://github.com/brainboxdotcc/DPP/blob/ced36fd7652ed24ec3784cbc150722a1e5f6456a/include/dpp/coro/job.h#L48
> 
> https://github.com/brainboxdotcc/DPP/blob/ced36fd7652ed24ec3784cbc150722a1e5f6456a/include/dpp/coro/async.h#L103
> 
> https://github.com/brainboxdotcc/DPP/blob/ced36fd7652ed24ec3784cbc150722a1e5f6456a/library/CMakeLists.txt#L292
> 
> (just remove the word "experimental" here, keep the `message()`).
> https://github.com/brainboxdotcc/DPP/blob/ced36fd7652ed24ec3784cbc150722a1e5f6456a/include/dpp/coro/task.h#L87
> 
> https://github.com/brainboxdotcc/DPP/blob/ced36fd7652ed24ec3784cbc150722a1e5f6456a/include/dpp/coro/coroutine.h#L67

should I just remove these comments altogether?

**Review by Jaskowicz1 — APPROVED** [review]

**Pull request merged**